services/net/content/ContentManager.cs

```diff
@@ -385,7 +385,7 @@ private async Task<ContentModel> CopyFileWithSSHAsync(DataLocationModel dataLoca
         Stream? file = null;
         try
         {
-            var fullPath = path.CombineWith(model.FilePath.MakeRelativePath());
+            var fullPath = path.CombineWith(model.FilePath.MakeRelativePath()).Replace("~/", $"{client.WorkingDirectory}/");
```

Comment:
If the connection is a relative path for the current user, it will change this so that the client can find the file.  For some reason the SSH client doesn't know what a `~` is.